File: 프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.java
class Solution {
    public int solution(int a, int b) {
        int ret = 0;
        if(a % 2 != 0 && b % 2 != 0) ret = a*a + b*b;
        else if(a % 2 == 0 && b % 2 == 0) ret = Math.abs(a - b);
        else ret = 2 *(a + b);
        return ret;
    }
}

File: 프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.py
def solution(a, b):
    if a % 2 == 0 and b % 2 == 0:
        return abs(a-b)
    elif a % 2 == 1 and b % 2 == 1:
        return a**2 + b**2
    else:
        return 2 *(a+b)